_data/providers.json

```diff
@@ -256,7 +256,8 @@
     {
       "name": "MC-HOST24.de",
       "url": "https://mc-host24.de/",
-      "description_template": "default"
+      "description_template": "default",
+      "description": "Make sure your remote address is 'auto' and your bedrock address is your server IP."
```

Comment:
What about the port? Similar to the other instructions; please include details about the port - Feel free to copy & adapt instructions from another host.

Reply:
Ah, makes sense - but nope, as soon as you want to edit something you basically have to write the whole thing :/